views/food.erb

```diff
@@ -0,0 +1,78 @@
+<html>
```

Comment:
Watch out for indentation when you use nested tags, once you start nesting, each subsequent tag should be indented one level. so here with the body and then on lines 4,5,7,etc.